x/wasm/alias.go

```diff
@@ -54,6 +55,11 @@ var (
 	NewContract               = types.NewContract
 	CosmosResult              = types.CosmosResult
 
+	// genesis aliases
+	ValidateGenesis = types.ValidateGenesis
```

Comment:
Is this the new style to place commands?

I feel "best practices" on the cosmos sdk changes every couple months and I cannot keep up

Reply:
No, I just wanted `InitGenesis` to be in keeper folder, so it can use unexposed functions `setContractInfo` and `setContractInfo`.  Is there a better way to access them from the outer package without exposing those functions?

Reply:
Good reason. And I am happy to have those functions unexported